### DISCUSSION TITLE
DONE: December 10th: Madison Clojure End of Year Celebration

### DISCUSSION THREAD

**Discussion by frenchy64** [madclj/madclj.com#21] [Announcements]
👍 to RSVP

https://madclj.com/

Wednesday, December 10th, 2025
6:30 PM to 9:00 PM CT

[StartingBlock Madison](https://www.google.com/maps/search/?api=1&query=43.081245%2C%20-89.374054)
821 E Washington Ave 2nd floor · Madison, WI

Let's eat, hang out and celebrate another year of Madison Clojure.
We'll have a loose meetup structure to help make the most of our time together.

Plan:
- 6:30pm-7pm: arrive, pizza will be available. comment below if you need to be let in.
- 7pm-9pm: meetup

Topics:
- personal achievements you'd like to share
- exciting Clojure developments in 2025
- Clojure topics for 2026?

**Comment by frenchy64** [comment]
@madclj/meetup our Madison Clojure end-of-year celebration is scheduled for Wednesday December 10th. We'll have a loosely structured meetup, hope to see you there!

[RSVP](https://github.com/orgs/madclj/discussions/21)

**Comment by ericnormand** [comment]
It will be just me (no kids).

**Comment by brunchboy** [comment]
I would love to come and catch up with people, but this is when Chris’ mom chose to take us to dinner to hear about our Southeast Asia vacation (and the weather there was much nicer than what we came back to here, I have to say). Hopefully next time!

**Reply by frenchy64** [reply]
@brunchboy will you be in Madison for this? https://scicloj.github.io/clojure-jam-2026/

Maybe we could do a hybrid presentation, where we live stream a talk from Madison with a live audience. I might propose something, you might want to consider too!

**Comment by jkirschnik12** [comment]
I think I'm getting sick so I won't be able to a make it tonight, hopefully I can make it to the next one!

**Reply by frenchy64** [reply]
2026 schedule is up! https://madclj.com/

**Comment by frenchy64** [comment]
Pizza is ordered, I'll be at the door at 6:30pm.

**Comment by frenchy64** [comment]
Notes:
- lots of exciting personal achievements, reach out to attendees to find out and congratulate them!
- exciting 2025 Clojure stuff
  - Madison representation @ conj
  - cljr port of Clojure CLI
  - in progress porting SCI to clojure clr
  - Timothy Pratley's thing (TBD)
  - porting google closure's test suite to cljs
- 2026
  - more conference live streams
    - Clojure Jam 2026
  - run a conference?
  - TBD request from @devn for @dpetranek to present on LLMs + OWL (needs clarification)
  - cowork!!
